html/assets/machinery.js

```diff
@@ -14,26 +14,26 @@ $(document).ready(function () {
     "patterns",
     "repositories",
     "services"
-  ]
+  ];
 
 
   // Enrich description with meta information summaries
-  description.meta_info = {}
+  description.meta_info = {};
   $.each(scopes, function(index, scope) {
     if(description.meta[scope]) {
       description.meta_info[scope] = " (" +
         "inspected host: '" + description.meta[scope].hostname + "', " +
-        "at: " + new Date(description.meta[scope].modified).toLocaleString() + ")"
+        "at: " + new Date(description.meta[scope].modified).toLocaleString() + ")";
     }
-  })
+  });
 
   $.each(scopes, function(index, scope) {
-    templates[scope] = Hogan.compile($("#scope_" + scope).html())
-  })
-  template = Hogan.compile($('#content').html())
+    templates[scope] = Hogan.compile($("#scope_" + scope).html());
+  });
+  template = Hogan.compile($('#content').html());
```

Comment:
Mixed double and single quotes.<br>'template' is not defined.<br>'Hogan' is not defined.<br>Too many errors. (21% scanned).